src/awkward/_connect/rdataframe/from_rdataframe.py

```diff
@@ -63,7 +63,7 @@
 assert done is True
 
 
-def from_rdataframe(data_frame, columns, offsets_type="int64_t"):
+def from_rdataframe(data_frame, columns, offsets_type="int"):
```

Comment:
Our interfaces should not use any platform-dependent types. Our default integer type is `int64_t`. (Same as #2199.)

Reply:
thanks! reverted.